charts/incubator/frigate/questions.yaml

```diff
@@ -1097,7 +1093,6 @@ questions:
                         Shadow - Shadow for font
                       schema:
                         type: string
-                        required: true
```

Comment:
Enums will always have a value. You can add an extra pair of value/decription like
`value: ""`
`description: Disable`
Then it will have `""` value